nominees/bigbluebutton.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "BigBlueButton",
+  "description": "BigBlueButton is an open source web conferencing system. It supports real-time sharing of audio, video, slides (with whiteboard controls), chat, and the screen. Instructors can engage remote students with polling, emojis, multi-user whiteboard, and breakout rooms. The educational use cases for BigBlueButton are online tutoring (one-to-one), flipped classrooms (recording content ahead of your session), group collaboration (many-to-many), online classes (one-to-many)",
+  "website": "https://bigbluebutton.org/",
+  "license": [
+    {
+      "spdx": "LGPL-3.0",
+      "licenseURL": "https://github.com/bigbluebutton/bigbluebutton/blob/develop/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 4,
+      "evidenceURL": "https://bigbluebutton.org/2020/04/19/bigbluebutton-at-orange-county-public-schools/"
+    }
+  ],
+  "sectors": [],
```

Comment:
@rachita19082 could you please include `Education` to the sectors array as a string entry. Thanks.

Reply:
@nathanbaleeta Updated in commit 350e382

Reply:
@nathanbaleeta Updated in commit 6933ac0